ConsoleApp1/Ecosystem/Biotic/Animal.cs

```diff
@@ -94,6 +94,25 @@ virtual public void StayWithAlpha()
             Follow(myAlpha.x, myAlpha.y);
         }
 
+        protected Plant FindClosestPlant(World world) {
+            double closestPreyDistance = world.sizeX * world.sizeY;
+            Plant closestPrey = null;
+            for (int j = 0; j < world.plants.Count(); j++) {
+                Plant prey = world.plants[j];  // TODO: out of range exception?
```

Comment:
I would vote for using a foreach loop here. We don't need the "j" counter and we will avoid possible "out of range" exception.

Reply:
`foreach (Plant prey : world.plants)   {/*...*/}`